package.json

```diff
@@ -4,15 +4,13 @@
   "author": "Ian Mitchell",
   "description": "TODO",
   "license": "MIT",
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
-  "module": "dist/mylib.esm.js",
-  "type": "module",
```

Comment:
I think we want to leave this in - my opinion ended up being that we should only target ESM / Modules